styles/arch-wiki/catppuccin.user.css

```diff
@@ -108,356 +135,99 @@
-    }
-
-    .mw-headline::before {
-      color: @accent-color !important;
+    
+    .mw-search-profile-tabs {
+        background-color: @base !important;
+        color: @text !important;
+        border-color: @base !important;
     }
-
-    .cdx-button {
-      background-color: @base !important;
-      color: @text !important;
-      border-color: @base !important;
+    
+    .cdx-search-input {
+        border-color: @mantle !important;
     }
 
-    .mw-body {
-      background-color: @base !important;
-      color: @text !important;
-      border-color: @base !important;
+    .oo-ui-widget, .oo-ui-widget:hover
+    .oo-ui-inputWidget-input {
+        background-color: @surface0 !important;
+        color: @text !important;
     }
 
-    .mw-search-profile-tabs {
-      background-color: @base !important;
-      color: @text !important;
-      border-color: @base !important;
+    .oo-ui-inputWidget-input {
+        background-color: @surface0 !important;
+        color: @text !important;
+        border-color: @surface0;
     }
-
+    
     .mw-content-ltr {
-      background-color: @base !important;
+        background-color: @base !important;
     }
 
     .vector-sticky-pinned-container::after {
-      background: @base !important;
+        background: @base !important;
     }
 
-    .cdx-search-input {
-      border-color: @mantle !important;
+    /* template box */
+    .archwiki-template-box {
+        background-color: @mantle !important;
+        color: @text !important;
+        border-color: @mantle !important;
     }
-
-    .oo-ui-widget:hover {
-      background-color: @surface0 !important;
+    
+    /* table */
+    .wikitable, tr, td, th {
+      background-color: @mantle !important;
+      color: @text !important;
+      border-color: @crust !important;
+    }
+
+    /* code */
+    code {
+        background-color: @mantle !important;
+        color: @text !important
+    }
+      
+    h2 > .mw-headline::before,
+    h3 > .mw-headline::before,
+    h4 > .mw-headline::before,
+    h5 > .mw-headline::before,
+    h6 > .mw-headline::before {
+        color: @accent-color;
+    }
+    
+    /* languages */
+    pre {
+        background-color: @crust !important;
+        border-color: @crust !important;
+        color: @text !important;
+    }
+    
+    /* buttons */
+    .btn, button, .vector-pinnable-header-toggle-button, .cdx-button:enabled {
+        background-color: @accent-color !important;
+        color: @base;
+    }
+    
+    /* popup */
+    .oo-ui-popupWidget-popup {
+         background-color: @crust !important;
+         color: @text;
+    }
+    
+    /* make icons bright when the flavor isn't latte */
+    & when not (@lookup = latte) {
+        label, .oo-ui-icon-search, .cdx-text-input__icon.cdx-text-input__start-icon, #mw-indicator-mw-helplink, .mw-ui-icon-wikimedia-bright, .oo-ui-indicator-clear {
+            filter: brightness(0) invert(1) !important;
+        }
```

Comment:
Is this an `<img>` element?

Reply:
no?

Reply:
What is it then? Why are we filtering it?

Reply:
the element uses ``background-image``